Revert/Sources/TransformLayersViewController.swift

```diff
@@ -13,9 +13,11 @@ final class TransformLayersViewController: RevertViewController {
 
   override func viewWillLayoutSubviews() {
     super.viewWillLayoutSubviews()
+    
+    let fortyFiveDegreesInRadian = CGFloat(GLKMathDegreesToRadians(10))
```

Comment:
`tenDegreesInRadians` not `fortyFiveDegreesInRadian?  Actually the variable/constant name should not reference the value stored.  So just `rotationInRadians` or something would be better.